webrtc.html

```diff
@@ -8942,6 +8956,52 @@ <h2 id="sec.stats-model">Statistics Model</h2>
       [[!WEBRTC-STATS]].</p>
     </section>
     <section>
+      <h3>The stats selection algorithm</h3>
+      <p>The <dfn>stats selection algorithm</dfn> is as follows:</p>
+      <ol>
+        <li>
+          Let <var>result</var> be an empty RTCStatsReport.
+        </li>
+        <li>If <var>selector</var> is <code>null</code>, gather stats for the
+          whole <var>connection</var>, add them to <var>result</var>, return
+          <var>result</var>, and abort these steps.
+        </li>
+        <li>If <var>selector</var> is an <a>RTCRtpSender</a>, gather stats for
+          and add the following objects to <var>result</var>:
+          <ul>
+            <li>
+            All <code>RTCInboundRTPStreamStats</code> objects corresponding to
+            <var>selector</var>.
+            </li>
+            <li>
+            All stats objects referenced directly or indirectly by the
+            <code>RTCInboundRTPStreamStats</code> objects added.
+            </li>
+          </ul>
+        </li>
+        <li>If <var>selector</var> is an <a>RTCRtpReceiver</a>, gather stats
+          for and add the following objects to <var>result</var>:
+          <ul>
+            <li>
+            All <code>RTCOutboundRTPStreamStats</code> objects corresponding
+            to <var>selector</var>.
+            </li>
+            <li>
+            All stats objects referenced directly or indirectly by the
+            <code>RTCOutboundRTPStreamStats</code> added.
+            </li>
+          </ul>
+        </li>
+        <li>
+          Return <var>result</var>.
+        </li>
+      </ol>
+      <p>
+        A stats object is said to "correspond to" a selector if its "ssrc" stats
+        attribute matches an <code><a>ssrc</a></code> in the selector.
```

Comment:
This needs a bit more language - SSRC needs to be one of the encodings on the sender, and needs to not match across transports.